Version 3.1

eriklacson opened the pull request:
Hey Benj, Quick favor. Please merge to master and deploy to prod. Thanks!

eriklacson committed:
corrected field names in cofinanced projects

eriklacson committed:
added field mapping spreadsheet for OIST reference

benjdayco commented:
Oops. Already on my way home and I left my MacBook at adb. I'll do it first
thing tomorrow.

On May 15, 2017 4:21 PM, "Erik Lacson" <notifications@github.com> wrote:

Hey Benj, Quick favor. Please merge to master and deploy to prod. Thanks!
------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/benjdayco/CIS/pull/11
Commit Summary

   - corrected field names in cofinanced projects
   - added field mapping spreadsheet for OIST reference

File Changes

   - *A* FieldMapping/Mapping of Fields New Web Design.xlsx
   <https://github.com/benjdayco/CIS/pull/11/files#diff-0> (0)
   - *M* cofinance-projects.html
   <https://github.com/benjdayco/CIS/pull/11/files#diff-1> (2)

Patch Links:

   - https://github.com/benjdayco/CIS/pull/11.patch
   - https://github.com/benjdayco/CIS/pull/11.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/benjdayco/CIS/pull/11>, or mute the thread
<https://github.com/notifications/unsubscribe-auth/ASe2pA0dHgd6eSXgMGYoFSCHzXLsGLx7ks5r6ArxgaJpZM4Nazqa>
.

eriklacson commented:
Oks lang. No biggie.

On May 15, 2017 4:36 PM, "benjdayco" <notifications@github.com> wrote:

Oops. Already on my way home and I left my MacBook at adb. I'll do it first
thing tomorrow.

On May 15, 2017 4:21 PM, "Erik Lacson" <notifications@github.com> wrote:

Hey Benj, Quick favor. Please merge to master and deploy to prod. Thanks!
------------------------------
You can view, comment on, or merge this pull request online at:

https://github.com/benjdayco/CIS/pull/11
Commit Summary

- corrected field names in cofinanced projects
- added field mapping spreadsheet for OIST reference

File Changes

- *A* FieldMapping/Mapping of Fields New Web Design.xlsx
<https://github.com/benjdayco/CIS/pull/11/files#diff-0> (0)
- *M* cofinance-projects.html
<https://github.com/benjdayco/CIS/pull/11/files#diff-1> (2)

Patch Links:

- https://github.com/benjdayco/CIS/pull/11.patch
- https://github.com/benjdayco/CIS/pull/11.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/benjdayco/CIS/pull/11>, or mute the thread
<https://github.com/notifications/unsubscribe-auth/
ASe2pA0dHgd6eSXgMGYoFSCHzXLsGLx7ks5r6ArxgaJpZM4Nazqa>
.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/benjdayco/CIS/pull/11#issuecomment-301411864>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AD2TbC4SuE9wZQ3mhyhayDQDGQJPMWtYks5r6A6lgaJpZM4Nazqa>
.

benjdayco commented:
Done!
